### ISSUE TITLE
Multiple issues  (Fedora Silverblue 41)

### ISSUE THREAD

**Issue by GitGrazer** [dracula/gtk#294]
Hello, first of all - thank you for making this awesome theme! I do have some issues with it: 

OS - Fedora Silverblue 41 (+latest updates)
Applied the flatpak theme using [https://github.com/refi64/stylepak](url)

Some flatpak applications appear to have squished menus (no padding) and there's some artifacts in Transmission. 

![Screenshot From 2024-11-02 20-59-43](https://github.com/user-attachments/assets/8e056c1d-6fb5-46cd-a597-65857e8efeb5)

![Screenshot From 2024-11-02 21-01-15](https://github.com/user-attachments/assets/481efbcc-0449-4389-9b95-99f6de231f55)

![Screenshot From 2024-11-02 21-01-47](https://github.com/user-attachments/assets/1f248290-3025-4989-a980-7749fed2dbff)

![Screenshot From 2024-11-02 21-03-58](https://github.com/user-attachments/assets/55f8de03-4dc5-4bb8-8a42-840a207140c3)

**Comment by EliverLara** [comment]
Hi, did you follow this step?

> **For the latest versions of Gnome using Gtk4(libadwaita) some extra steps should be done in order to get it working properly, you could do it in two ways:**

> 1.- Manual way
  > - Copy the `assets` folder to the `~/.config` directory
  > - Copy the `gtk-4.0/gtk.css` and `gtk-4.0/gtk-dark.css` files to `~/.config/gtk-4.0/`

> 2.- Using [this script](https://github.com/odziom91/libadwaita-theme-changer) which will copy the needed folders to the right directories

**Comment by GitGrazer** [comment]
> Hi, did you follow this step?
> 
> > **For the latest versions of Gnome using Gtk4(libadwaita) some extra steps should be done in order to get it working properly, you could do it in two ways:**
> 
> > 1.- Manual way
> > 
> > * Copy the `assets` folder to the `~/.config` directory
> > * Copy the `gtk-4.0/gtk.css` and `gtk-4.0/gtk-dark.css` files to `~/.config/gtk-4.0/`
> 
> > 2.- Using [this script](https://github.com/odziom91/libadwaita-theme-changer) which will copy the needed folders to the right directories

Hi, thanks for the reply.

Yes, I did both the manual and the script method and GTK Theme Adapter tool. The theme works fine   with libadwaita, it's just the flatpaks that are are broken.

**Comment by EliverLara** [comment]
Have you provide permissions to flatpak access your themes and config directory?

**Comment by GitGrazer** [comment]
> Have you provide permissions to flatpak access your themes and config directory?

Yes, obviously. As you can see, the theme is applied on the flatpak apps, it just looks wrong. The Orchis theme also had the same issue for a while, but it was fixed recently.

**Comment by GitGrazer** [comment]
No, solution to this issue? I've been trying the new updated version to work with flatpaks, but the same problem persists.

**Comment by EliverLara** [comment]
@GitGrazer Hi, I'm sorry but I haven't been able to reproduce the issue, I also use fedora and it looks fine on my side:

![Image](https://github.com/user-attachments/assets/25d78368-3073-4a7a-9ba2-e56c6d5974a8)

![Image](https://github.com/user-attachments/assets/1bbd0b2a-94f5-4831-b2b9-31c94cb8c836)

Could you try using those commands, please?

```
sudo flatpak override --filesystem=~/.themes/

sudo flatpak override --filesystem=xdg-config/gtk-4.0

sudo flatpak override --filesystem=xdg-config/assets
```

**Comment by GitGrazer** [comment]
Thanks you for the reply! The flatpak apps look okay now :)

I was missing the sudo flatpak override --filesystem=xdg-config/assets permission. 

Artifacting in Transmision, Gnome System Monitor and Nautilus/Files still remain though. 

Thanks for working on the awesome theme!
